lib/matplotlib/sankey.py

```diff
@@ -780,8 +781,9 @@ def _get_angle(a, r):
         # where either could determine the margins (e.g., arrow shoulders).
 
         # Add this diagram as a subdiagram.
-        self.diagrams.append(Bunch(patch=patch, flows=flows, angles=angles,
-                                   tips=tips, text=text, texts=texts))
+        self.diagrams.append(types.SimpleNamespace(patch=patch, flows=flows,
+                                                   angles=angles, tips=tips,
+                                                   text=text, texts=texts))
```

Comment:
As illustrated here, `types.SimpleNamespace` is a lot more characters to type than `Bunch`, and the pep8 line length limit is quite small. Just a thought: how about `from types import SimpleNamespace as Bunch`?

Reply:
I don't see the need for a shortcut like that. There are other ways to wrap it, such as the one I just pushed.